ide/vscode/launch-raspberrypi-swd.json

```diff
@@ -8,15 +8,15 @@
             "request": "launch",
             "type": "cortex-debug",
             "servertype": "openocd",
-            // This may need to be arm-none-eabi-gdb depending on your system
+            // This may need to be arm-none-eabi-gdb if VSCode is not running on a Raspberry Pi
             "gdbPath" : "gdb-multiarch",
             "device": "RP2040",
             "configFiles": [
                 "interface/raspberrypi-swd.cfg",
```

Comment:
AIUI, this is for when the SWD pins on a Pico are connected directly to the GPIO pins on a Raspberry Pi (see page 19 of https://datasheets.raspberrypi.com/pico/getting-started-with-pico.pdf ), which doesn't match your "A target connected via another Pico" assumption in the README. I.e. you might want to clarify your README a bit further?

Reply:
Ah - didn't know that: thanks
Yes I'll tweak the README

Reply:
@lurch comment really clarified my understanding of what's going on in the different configurations, so apologies for the further changes but I hope 27a065 is an improvement.